### PR TITLE
🌱 Move controllers to internal - part3

### PR DESCRIPTION
**Pull request by fabriziopandini**
**What this PR does / why we need it**:
This PR moves the last group of controllers from /controllers to internal/controllers

**Which issue(s) this PR fixes**:
Rif #5455

### CODE REVIEW

File: controllers/alias.go [hunk 1]
```diff
@@ -24,10 +24,14 @@ import (
 
 	"sigs.k8s.io/cluster-api/controllers/remote"
 	clustercontroller "sigs.k8s.io/cluster-api/internal/controllers/cluster"
+	clusterclasscontroller "sigs.k8s.io/cluster-api/internal/controllers/clusterclass"
 	machinecontroller "sigs.k8s.io/cluster-api/internal/controllers/machine"
 	machinedeploymentcontroller "sigs.k8s.io/cluster-api/internal/controllers/machinedeployment"
 	machinehealthcheckcontroller "sigs.k8s.io/cluster-api/internal/controllers/machinehealthcheck"
 	machinesetcontroller "sigs.k8s.io/cluster-api/internal/controllers/machineset"
+	clustertopologycontroller "sigs.k8s.io/cluster-api/internal/controllers/topology/cluster"
+	machinedeploymentTopologycontroller "sigs.k8s.io/cluster-api/internal/controllers/topology/machinedeployment"
```
Comment (sbueringer) [inline controllers/alias.go:33]:
```suggestion
	machinedeploymenttopologycontroller "sigs.k8s.io/cluster-api/internal/controllers/topology/machinedeployment"
```
nit

**Review by sbueringer — COMMENTED** [review]
Great work! One nit, otherwise lgtm

**Review by vincepri — APPROVED** [review]
/approve

**Pull request merged**